_docs/welcome/feature-stages.md

```diff
@@ -29,94 +28,75 @@ within the product, not to the product as a whole. Here is a high level descript
 |  **Performance**         | Not quantified or guaranteed     | Not quantified or guaranteed         | Perf (latency/scale) is quantified, documented, with guarantees against regression 
 |   **Deprecation Policy**        | None     | Weak - 3 months         | Dependable,  Firm. 1 year notice will be provided before changes
 
-## Istio 0.3.0 features status
+## Istio 0.4 features
 
 Below is our list of existing features and their current phases. This information will be updated after every monthly release.
 
-### Config / APIs
-
-
-| Feature           | Phase      
-|-------------------|-------------------
-| [Mesh-wide Settings](https://istio.io/docs/reference/config/service-mesh.html) | Alpha
-| [Basic config validation](https://github.com/istio/istio/issues/1894)    | Alpha
-
-
-### Networking
+### Traffic Management
 
 | Feature           | Phase      
 |-------------------|-------------------
-| [Protocols HTTP 1.1](https://github.com/istio/istio/blob/master/pilot/model/service.go#L104)         | Beta
-| [Distributed Tracing Zipkin / Jaeger](https://istio.io/docs/tasks/telemetry/distributed-tracing.html)        | Beta
-| [Ingress Controller](https://istio.io/docs/tasks/traffic-management/ingress.html)         | Alpha
-| [Basic Egress](https://istio.io/docs/tasks/traffic-management/egress.html)               | Alpha
-| [Protocols HTTP 2.0](https://github.com/istio/istio/blob/master/pilot/model/service.go#L101)        | Alpha
-| [Protocols gRPC](https://github.com/istio/istio/blob/master/pilot/model/service.go#L97)     		   | Alpha
-| [Protocols TCP](https://github.com/istio/istio/blob/master/pilot/model/service.go#L107)         	   | Alpha
+| [Memquota Implementation and Integration](https://istio.io/docs/tasks/telemetry/metrics-logs.html) | Beta
+| [Protocols: HTTP 1.1](https://github.com/istio/istio/blob/master/pilot/model/service.go#L104)  | Beta
+| [Protocols: HTTP 2.0](https://github.com/istio/istio/blob/master/pilot/model/service.go#L101)  | Alpha
+| [Protocols: gRPC](https://github.com/istio/istio/blob/master/pilot/model/service.go#L97)    | Alpha
+| [Protocols: TCP](https://github.com/istio/istio/blob/master/pilot/model/service.go#L107)     | Alpha
```

Comment:
Linking to code is not good. No one cares :).
I would have just two:
HTTP1.1/Http2/gRPC support - Beta
TCP proxy support - Beta

Reply:
I disagree that gRPC, HTTP2 and TCP protocols are in beta. We need more testing, and as you rightfully pointed out, actual docs. And you are correct, we need user docs and not links to code. If your team has bandwidth to help write/improve docs, we would greatly appreciate that.

Reply:
Sure, we can move it to alpha. The point is linking to code is pointless. And there really is no task to demonstrate http1 vs http2. Its equivalent to "providing" Istio works with HTTP2. A better option would be to change one of bookinfo services to use HTTP2.

Reply:
Agree with not linking to code, these should only link to istio.io docs. Fine to have some without docs and some with for now, especially things in alpha. I think Beta requirement is that it has a doc to link to, so we can't mark anything beta if it doesn't have a doc link :)